app/models/sidebar_url.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SidebarUrl < ActiveRecord::Base
+  validates :name, presence: true
+  validates :value, presence: true
```

Comment:
Is there a reason we are not validating the value on the server side? The server side routes and ember routers should map 1 to 1. If we don't validate the routes on the server side, we may risk ending up with invalid values being stored in the database.